ui/raidboss/data/04-sb/raid/o7s.txt

```diff
@@ -2,100 +2,100 @@
-332 "Diffractive Plasma" sync /:Guardian:278B:/
-344 "Diffractive Plasma" sync /:Guardian:278B:/
-356 "Prey" sync /:Guardian:278A:/
-366 "Viral Weapon" sync /:Guardian:277C:/
+319 "Magnetism/Repel" sync / 1[56]:Fire Control System:2779:/
+332 "Diffractive Plasma" sync / 1[56]:Guardian:278B:/
+344 "Diffractive Plasma" sync / 1[56]:Guardian:278B:/
+356 "Prey" sync / 1[56]:Guardian:278A:/
+366 "Viral Weapon" sync / 1[56]:Guardian:277C:/
 367 "Temporary Misdirection" duration 15
-372 "Magitek Ray" sync /:Guardian:2788:/
-375 "--untargetable--" sync /:Guardian:2937:/
+372 "Magitek Ray" sync / 1[56]:Guardian:2788:/
+375 "--untargetable--" sync / 1[56]:Guardian:2937:/
 382 "Chain Cannon" duration 2
-388 "Main Cannon" sync /:Guardian:2790:/
+388 "Main Cannon" sync / 1[56]:Guardian:2790:/
 389 "Radar" duration 2
-391 "--targetable--" sync /:Guardian:2938:/
-400 "Arm And Hammer" sync /:Guardian:2789:/
-408 "Diffractive Plasma" sync /:Guardian:278B:/
+391 "--targetable--" sync / 1[56]:Guardian:2938:/
+400 "Arm And Hammer" sync / 1[56]:Guardian:2789:/
+408 "Diffractive Plasma" sync / 1[56]:Guardian:278B:/
 
 # Ultros
-416 "Paste Ultros" sync /:Guardian:2776:/
-417 "Ink" sync /:Guardian:277D:/
-424 "Tentacle Simulation" sync /:Guardian:275E:/
-430 "Bomb Deployment" sync /:Guardian:2762:/
-439 "Atomic Ray" sync /:Guardian:278D:/
-444 "Magitek Ray" sync /:Guardian:2788:/
-453 "Diffractive Plasma" sync /:Guardian:278B:/
-463 "Run Ultros (SW)" sync /:Guardian:276F:/
-469 "Missile Simulation" sync /:Guardian:2764:/
+416 "Paste Ultros" sync / 1[56]:Guardian:2776:/
+417 "Ink" sync / 1[56]:Guardian:277D:/
+424 "Tentacle Simulation" sync / 1[56]:Guardian:275E:/
+430 "Bomb Deployment" sync / 1[56]:Guardian:2762:/
+439 "Atomic Ray" sync / 1[56]:Guardian:278D:/
+444 "Magitek Ray" sync / 1[56]:Guardian:2788:/
+453 "Diffractive Plasma" sync / 1[56]:Guardian:278B:/
+463 "Run Ultros (SW)" sync / 1[56]:Guardian:276F:/
+469 "Missile Simulation" sync / 1[56]:Guardian:2764:/
 472 "Interrupt Stoneskin" duration 4
-474 "--untargetable--" sync /:Guardian:2937:/
+474 "--untargetable--" sync / 1[56]:Guardian:2937:/
 481 "Chain Cannon" duration 2
-487 "Main Cannon" sync /:Guardian:2790:/
-490 "--targetable--" sync /:Guardian:2937:/
-499 "Diffractive Plasma" sync /:Guardian:278B:/
+487 "Main Cannon" sync / 1[56]:Guardian:2790:/
+490 "--targetable--" sync / 1[56]:Guardian:2937:/
+499 "Diffractive Plasma" sync / 1[56]:Guardian:278B:/
 
 # Divergence point
-509 "Load/Skip ???" sync /:Guardian:(275C|2773):/
-510 "--sync--" sync /1A:........:Guardian gains the effect of Bibliotaph Simulation/ jump 2510       # Bibliotaph buff, jump to A1
-510 "--sync--" sync /1A:........:Guardian gains the effect of Dadaluma Simulation/ jump 3510         # Dadaluma buff, jump to A2
+509 "Load/Skip ???" sync / 1[56]:Guardian:(275C|2773):/
+510 "--sync--" sync / 1A:.{0,4}:Bibliotaph Simulation:[^:]+:.{8}:[^:]+:.{8}:Guardian:/ jump 2510       # Bibliotaph buff, jump to A1
```

Comment:
For robustness into the future, can I suggest converting these to not be so specific about the fields themselves? 

```suggestion
510 "--sync--" sync / 1A:[^:]*:Bibliotaph Simulation:[^:]*:[^:]*:[^:]*:[^:]*:Guardian:/ jump 2510       # Bibliotaph buff, jump to A1
```

Alternatively, maybe dig up the id so it can just be `1A:id:`?

Reply:
Sure, we can do that.

[Bibliotaph Simulation](https://xivapi.com/Status/1492?pretty=true)
[Dadaluma Simulation](https://xivapi.com/Status/1491?pretty=true)